### PR TITLE
add api doc for placement/placementdecision

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: apis/placement.json.adoc [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+[[_rhacm-docs_apis_placement_jsondefinitions]]
+== Definitions
+
+[[_rhacm-docs_apis_placement_jsonplacement]]
+=== Placement
+
+[options="header", cols=".^3a,.^4a"]
+|===
+|Name|Schema
```
Comment (dockerymick) [inline apis/placement.json.adoc]: Please add a description column for the Definitions

File: apis/placementdecision.json.adoc [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+[[_rhacm-docs_apis_placementdecision_jsondefinitions]]
+== Definitions
+
+[[_rhacm-docs_apis_placementdecision_jsonplacementdecision]]
+=== PlacementDecision
+
+[options="header", cols=".^3a,.^4a"]
+|===
+|Name|Schema
```
Comment (dockerymick) [inline apis/placementdecision.json.adoc]: Same suggestion here, please add a Description  column

File: apis/placement.json.adoc (hunk 1 above)
Comment (elgnay) [reply apis/placement.json.adoc]: Added.

File: apis/placementdecision.json.adoc (hunk 2 above)
Comment (elgnay) [reply apis/placementdecision.json.adoc]: Added.

**Comment by dockerymick** [comment]
@elgnay Thanks for your changes. I will review this PR shortly. I also created a [doc issue](https://github.com/open-cluster-management/backlog/issues/13762) for this PR so that I can track the progress

**Review by dockerymick — APPROVED** [review]
@elgnay thanks for adding the description column and for your updates. I made a couple of commits for consistency and added the files to the navigation so that the build can render. Moving forward to merge this

**Comment by dockerymick** [comment]
Related doc issue: https://github.com/open-cluster-management/backlog/issues/13762

**Pull request merged**